src/schema/workflow_execution_activity.yaml

```diff
@@ -293,7 +293,7 @@ classes:
         required: true
       id:
         structured_pattern:
-          syntax: "{id_nmdc_prefix}:calib-{id_shoulder}-{id_blade}{id_version}{id_locus}"
+          syntax: "{id_nmdc_prefix}:calib-{id_shoulder}-{id_blade}{id_version}$"
```

Comment:
This should not have version, will fix.

Reply:
Sorry, that was not a good place to leave that comment, I was referring about making WorkflowExecution abstract, not CalibrationInformation. This class seems like its not in the correct yaml, everything else in this yaml is for classes downstream of DataGeneration. Do you know the history on why this class is in this file and not nmdc.yaml?

Reply:
Oh yeah.....long story but the nmdc.yaml sources the workflow_execution.yaml but not vice versa.  The `CalibrationInformation` is the range of the `has_calibration` slot which is on some of the `WorkflowExecutions `(until we merge in Berkeley and apply some ingestions).  Once that has_calibration is fixed, we can move this class and associated slot definition to the nmdc.yaml.

`WorkflowExecution` is already abstract on berkeley fork, which makes total sense to me.